src/ServiceProvider.php

```diff
@@ -52,6 +52,7 @@ public function register(Application $app)
         $app['thumbnails.error_image'] = null;
         $app['thumbnails.cache_time'] = null;
         $app['thumbnails.limit_upscaling'] = true;
+        $app['thumbnails.restrict_alias'] = false;
```

Comment:
I think th normal user wont know what a route is, but he knows what an alias is. This options basically "restricts thumbnails to aliases" since the short name "restrict_alias". But like above, i can live with both.